src/libraries/System.Formats.Tar/ref/System.Formats.Tar.cs

```diff
@@ -13,6 +13,11 @@ public sealed partial class GnuTarEntry : System.Formats.Tar.PosixTarEntry
         public System.DateTimeOffset AccessTime { get { throw null; } set { } }
         public System.DateTimeOffset ChangeTime { get { throw null; } set { } }
     }
+    public sealed partial class PaxGlobalExtendedAttributesTarEntry : System.Formats.Tar.PosixTarEntry
+    {
+        public PaxGlobalExtendedAttributesTarEntry(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>> globalExtendedAttributes) { }
```

Comment:
I guess there's no conversion ctor because there's no analogous type in any other format, so all clones would also have been PaxGEA entries?

Reply:
That is correct, it does not make sense to convert a tar entry that only exists in PAX. Do you have concerns about the design?

Reply:
No, it's fine.  It's just not symmetric, so I was making sure my understanding matched reality.